src/cljc/rems/common/util.cljc

```diff
@@ -554,3 +554,8 @@
   (is (= [] (replace-key [] :a :b)))
   (is (= {:b 1} (replace-key {:a 1} :a :b)))
   (is (= {:a 1} (replace-key {:a 1} :b :c))))
+
+(defn in?
```

Comment:
We could use `contains?` and regular sets so we don't have to use this helper. The sets are rather idiomatic in Clojure and more so in REMS codebase so far.